### ISSUE TITLE
Misleading text echoed to console

### ISSUE THREAD

**Issue by lopippo** [tomschr/daps#692]
Greetings!

When I build my documentation, I have the following text that is echoed to the console, and I must say that it is misleading:

```
    Must remake target '/home/rusconi/devel/massxpert/development/doc/user-manual/build/.profiled/noprofile/.validate'.
if [[ -n "" ]]; then \
  ccecho "error" "Fatal error: The document contains XML errors:"; \
  echo -e ""; \
  echo "--------------------------------"; \
fi
if [[ -n "" ]]; then \
  ccecho "error" "Fatal error: The following tables contain errors:"; \
  echo -e ""; \
  echo "--------------------------------"; \
fi
if [[ -n "" ]]; then \
  ccecho "error" "The following IDs contain unwanted characters:"; \
  echo -e ""; \
  echo "--------------------------------"; \
fi
if [[ -n "" ]]; then \
  ccecho "error" "Fatal error: The following images are missing:"; \
  echo -e ""; \
  echo "--------------------------------"; \
fi
if [[ -n "" ]]; then \
  ccecho "warn" "Warning: Image names not unique, multiple sources available for the following images:"; \
  echo -e ""; \
  echo "--------------------------------"; \
fi
if [[ -n "" ]]; then \
  ccecho "error" "Document does not validate!"; \
  exit 1; \
fi
touch /home/rusconi/devel/massxpert/development/doc/user-manual/build/.profiled/noprofile/.validate
    Successfully remade target file '/home/rusconi/devel/massxpert/development/doc/user-manual/build/.profiled/noprofile/.validate'.

```

The file that is the cause is this:

/usr/share/daps/make/validate.mk:77 et sqq

Either there is an actual error and that error is not displayed (and the processing goes on normally, which is unexpected) or there is not an error and the text is misleading.

Not sure what is the right option above :-)

Could you help me understand the code, please ?

Most sincerely,
Filippo
(still addict to daps :-)

**Comment by fsundermeyer** [comment]
This has already been fixed with cd323896530c. Please use https://github.com/openSUSE/daps/releases/tag/3.3.2 for the new stab le release DAPS 3.3.2

**Comment by lopippo** [comment]
> This has already been fixed with [cd32389](https://github.com/openSUSE/daps/commit/cd323896530c44347ac19e8f2a554cf3b2aca4a5). Please use https://github.com/openSUSE/daps/releases/tag/3.3.2 for the new stab le release DAPS 3.3.2

Aknowledged, thank you!
Sincerely
Filippo
